custom_components/zwave_mqtt/binary_sensor.py

```diff
@@ -16,11 +30,41 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
     """Set up Z-Wave binary_sensor from config entry."""
 
     @callback
-    def async_add_binary_sensor(value):
+    def async_add_binary_sensor(values):
         """Add Z-Wave Binary Sensor."""
-        binary_sensor = ZWaveBinarySensor(value)
 
-        async_add_entities([binary_sensor])
+        sensors_to_add = []
+
+        if values.primary.type == ValueType.LIST:
+
+            # Handle special cases
+            # we convert some of the Notification values into it's own binary sensor
+            # https://github.com/OpenZWave/open-zwave/blob/master/config/NotificationCCTypes.xml
+            for item in values.primary.value["List"]:
+                if values.primary.index == 6 and item["Value"] == 22:
```

Comment:
Here we would like to use enums or similar from openzwavemqtt to avoid having to use the integers directly.

Reply:
Yes, but that will be another PR. I am going to create constants and enums in the main lib.

Reply:
added comment in the code to as reminder